packages/mwp-router/src/RouteLayout.jsx

```diff
@@ -25,7 +25,12 @@ class RouteLayout extends React.Component {
 							path={path}
 							exact={route.exact || false}
 							strict={route.strict || false}
-							render={props => <AsyncRoute {...props} route={route} />}
+							render={props =>
+								<AsyncRoute
+									{...props}
+									{...childProps}
```

Comment:
could you explain why this was added? I'm not familiar with childProps, is that a react-router thing?

Reply:
ah, this was actually something I was starting to sketch out in order to support injecting props into components attached to routes, but it wasn't working out and I didn't clean up - will fix!